modules/serverless-resolving-operator-upgrade-failure.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * /serverless/admin_guide/serverless-upgrades.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-resolving-operator-upgrade-failure_{context}"]
+= Resolving {ServerlessOperatorName} upgrade failure
+
+In some cases, such as when you do manual uninstalls and reinstalls, you might encounter an error when upgrading {ServerlessOperatorName}. In this situation, you need to manually reinstall {ServerlessOperatorName}.
+
+.Procedure
+
+. Identify the version of {ServerlessOperatorName} that was installed originally by searching in the {ServerlessProductName} Release Notes.
++
+For example, the error message during attempted upgrade contains this string:
++
+[source,yaml]
+----
+The installed KnativeServing version is v1.5.0.
+----
++
+In this case, the KnativeServing `MAJOR.MINOR` version is `1.5`, which is mentioned in the release notes for {ServerlessProductName} 1.26: `OpenShift Serverless now uses Knative Serving 1.5`.
+
+. Uninstall {ServerlessOperatorName} and all its install plans.
+
+. Manually install {ServerlessOperatorName} of version discovered in the first step. To do that, first create a `serverless-subscription.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: serverless-operator
+  namespace: openshift-serverless
+spec:
+  channel: stable
+  name: serverless-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  installPlanApproval: Manual
+  startingCSV: serverless-operator.v1.26.0
+----
+
+. Then, install the subscription:
++
+[source,terminal]
+----
+$ oc apply -f serverless-subscription.yaml
+----
+
+. Upgrade by manually approving the upgrade install plans as they appear.
```

Comment:
```suggestion
. Upgrade by manually approving the upgrade install plans as they appear in the terminal window.
```

Reply:
Added this but may not be correct - it seemed to need some clarification, but please ignore if not applicable.